diffrax/_autocitation.py

```diff
@@ -374,7 +381,11 @@ def _backsolve_rms_norm(adjoint):
 
 @citation_rules.append
 def _explicit_solver(solver, terms=None):
-    if not isinstance(solver, AbstractImplicitSolver) and not is_sde(terms):
+    if (
+        not isinstance(solver, AbstractImplicitSolver)
+        and not is_sde(terms)
+        and not issubclass(type(solver), AbstractSRK)
```

Comment:
Why ths extra check?

Reply:
Without it, if I ran `citation(solver=ShARK())` it would just cite explicit solvers (Hairer) and not ShARK itself.

Reply:
Ah, I see. In that case I think think maybe this check should be `if not isinstance(solver, (AbstractImplicitSolver, AbstractSRK, AbstractItoSolver, AbstractStratonovichSolver)) and not is_sde(terms)`?

(I think.)

Basically just trying to exclude all the non-explicit solvers in one go.

Reply:
Good, will do.